lib/engine/game/base.rb

```diff
@@ -652,6 +653,8 @@ def sell_shares_and_change_price(bundle)
         case self.class::SELL_MOVEMENT
         when :down_share
           bundle.num_shares.times { @stock_market.move_down(corporation) }
+        when :down_per_10
+          (10 * bundle.num_shares / entity.total_shares).to_i.times { @stock_market.move_down(corporation) }
```

Comment:
can you just do (bundle.percent / 10).to_i

Reply:
For selling 25% does bundle.num_shares return 2.5? Or 2? In the former case one could add a 'ceil: true' parameter to num_shares. Have done similar changes to handle the 5% shares of NdM in 18MEX.

Isn't the down_per_10 the normal behavior in 18xx? I don't remember 1856 have any special rules for that.

Reply:
Ah, now I remember? The "big one" have 5% shares that do not affect the price if sold one, but you need to sell two to drop it. Right.
In case of 18MEX you wont drop price if selling 2 5%, but 18MEX has special handling in the game title code.

Reply:
> For selling 25% does bundle.num_shares return 2.5? Or 2? In the former case one could add a 'ceil: true' parameter to num_shares. Have done similar changes to handle the 5% shares of NdM in 18MEX.
> 
> Isn't the down_per_10 the normal behavior in 18xx? I don't remember 1856 have any special rules for that.

I think it's the normal for 20 share companies in most 18xx games, but this is distinctly different from dropping once per share (in a 20 share company)

The intention is that it gets rounded down, so selling 25% would result in 2 drops